javascript/javascript1/week3/travel-time.js

```diff
@@ -0,0 +1,15 @@
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+  calculator: function () {
```

Comment:
Split this function out of the object, so that your code looks like this:
```js
function calculator(travelInformation) {
...
}

calculator(travelInformation);
```
You might give you function another meaningful name like `getTime`